src/types/matrix/accountData.ts

```diff
@@ -4,6 +4,7 @@ export enum AccountDataEvent {
   IgnoredUserList = 'm.ignored_user_list',
 
   CinnySpaces = 'in.cinny.spaces',
+  CinnyExploreServers = 'in.cinny.explore_servers',
```

Comment:
I think naming it `in.cinny.explore` is good, as the content has `servers` array and allows `content` to add more things outside just servers.